validate/__main__.py

```diff
@@ -9,8 +9,7 @@ def parse_args():
     parser.add_argument('path', help='one or more paths of files to validate', type=str, nargs='+')
     parser.add_argument('-t', '--type', help='input data type',
         choices=['file-input', 'file-bam', 'file-vcf', 'file-fasta', 'file-fastq',
-        'file-bed', 'file-py', 'directory-rw', 'directory-r'],
-        default='file-input')
```

Comment:
I think when 'file-input' is used it won't show the detected file type in the success message since it uses the value from the args. The only place I saw 'file-input' ( GENERIC_FILE_INPUT) used was to determine whether the detected file type matched the one passed to the function and throw an error. Then if it was the generic one it would ignore it anyway. So it's not used at all as far as I could see. 

I could keep it and just change the success message to the detected type like I have changed already.

Reply:
Right, it wasn't printing that in the success message; I think we may just want to keep the three options I have below and then let the default be `file`